### ISSUE TITLE
When using dataSet, only one pointer is displayed on the line chart.

### ISSUE THREAD

**Issue by HikaruTakemoto** [Abhinandan-Kushwaha/react-native-gifted-charts#610]
I am using dataSet to draw six lines on a line chart. I think that when you drag on the chart, points appear on the lines. However, points only appear on the first line.

  ```
	const dataSet: Array<DataSet> = [
		{ data: data1, color: "blue", dataPointsColor: "black" },
		{ data: data2, color: "green", dataPointsColor: "black" },
		{ data: data3, color: "orange", dataPointsColor: "black" },
		{ data: data4, color: "red", dataPointsColor: "black" },
		{ data: data5, color: "purple", dataPointsColor: "black" },
		{ data: data6, color: "pink", dataPointsColor: "black" },
	];

<LineChart
					adjustToWidth
					curved
					hideDataPoints
					height={100}
					overflowTop={15}
					maxValue={500000}
					dataSet={dataSet}
					initialSpacing={0}
					noOfSections={4}
					thickness={1}
					rulesType="solid"
					rulesColor="lightgray"
					getPointerProps={(index: PointerIndex) => {
						console.log(index);
						setActivePointerIndex(index.pointerIndex);
						setActivePointerX(index.pointerX);
					}}
					pointerConfig={{
						pointerStripColor: "black",
						pointerStripWidth: 2,
						pointer1Color: "blue",
						pointer2Color: "green",
						pointer3Color: "orange",
						radius: 4,
					}}
				/>
```

**Comment by HikaruTakemoto** [comment]
https://github.com/Abhinandan-Kushwaha/react-native-gifted-charts/assets/62002986/61fedd2a-cb2f-4be2-9f21-f666dbdee430

**Comment by vorn-dev-ni** [comment]
Same only 1 item or pointer is displayed

**Comment by Abhinandan-Kushwaha** [comment]
Hi @HikaruTakemoto 👋
Thanks for pointing it out. It has been fixed in version `1.4.11` 🎉

Now the pointer will appear on each line of the dataSet. If you want the pointers to be of different colors, you can pass the colors in an array using the prop `pointerColorsForDataSet`.

Please use the latest version of the library.

cc: @vorn-dev-ni
